NDR Mediathek funzt nicht mehr

jodo234 opened the pull request:
Moinsen, ich finde hier leider niergends eine Möglichkeit dich zu kontaktieren, daher versuche ich es nun auf diesem Weg :)
Ich habe bis vor kurzem noch das NDR Plugin in der Version 1.1.2 von Membrane genutzt. Seit einiger Zeit funzt das aber nicht mehr richtig, wenn man irgendeine Sendung/Serie auswählt, ist der Ordner dann einfach leer. Ich habe daraufhin dein Repo installiert und dann die Version 1.1.3. Aber auch damit sind alle Ordner in der Mediathek einfach leer... man sieht die Namen der Serien und Sendungen, wenn man die aber öffnet, um eine Folge zu starten, sind die Unterordner einfach leer :/

68000a committed:
libard 5.1.6
- Beim Hinzufügen zu Favoriten gingen die Metadaten verloren
- Menüpunkte "Neu" und "Meistgesehen" entfernt
- Thumbnail funktioniert wieder
- Kleinere Anpassungen für Python3

68000a committed:
Kleinere Anpassungen für Python3

68000a committed:
SWR Mediathek seit 19.08.2019 nicht mehr verfügbar

68000a committed:
SWR Mediathek seit 19.08.2019 nicht mehr verfügbar

68000a committed:
SWR Mediathek - 1.1.4
SWR-Mediathek komplett in die ARD-Mediathek umgezogen
siehe: https://www.swr.de/home/Umzug-der-SWR-Mediathek-in-die-ARD-Mediathek,artikel-mediathek-umzug-100.html

68000a committed:
WDR Mediathek - 1.2.3 / libwdr 1.2.5
- Beim Hinzufügen zu Favoriten gingen die Metadaten verloren
- Kleinere Anpassungen für Python3

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
Das Erste Mediathek 1.1.3 / libdaserste 1.1.4
- Beim Hinzufügen zu Favoriten gingen die Metadaten verloren
- Kleinere Anpassungen für Python3

68000a committed:
Kleinere Anpassungen für Python3

68000a committed:
NDR Mediathek seit 11.12.2019 nicht mehr verfügbar

68000a committed:
NDR Mediathek - 1.1.4
NDR-Mediathek komplett in die ARD-Mediathek umgezogen
siehe: https://www.presseportal.de/pm/6561/4464810

68000a committed:
libarte 1.0.14
Bei Sendungen in Originalsprache wurden fälschlicherweise fremdsprachige Untertitel angezeigt

68000a committed:
ZDF Mediathek - 4.1.3 / ZDFtivi Mediathek - 1.0.1 / libzdf 2.1.14
- Beim Hinzufügen zu Favoriten gingen die Metadaten verloren
- Kleinere Änderungen beim Zusammenbauen des Titelnamens
- Neue Logos

68000a committed:
libmediathek3 1.5.11
Unterstützung für Python3

68000a committed:
Schönheitskorrekturen

68000a committed:
libmediathek3 1.5.12
Untertitelte Sendungen konnten u.U. nicht abgespielt werden

68000a committed:
Schönheitskorrektur: Im Plot zur Sendung werden jetzt die HTML-Entities berücksichtigt

68000a committed:
libmediathek3 1.5.13
Kleine Anpassung für die Neuimplementierung der Suche in der ARD-Mediathek

68000a committed:
ARD Mediathek - 5.1.2 / libard 5.1.7
Suche ist wieder verfügbar

68000a committed:
libard 5.1.8
Sendungen nach A-Z funktioniert wieder

68000a committed:
README-Datei aktualisiert

68000a committed:
README-Datei aktualisiert

68000a committed:
README-Datei aktualisiert

68000a committed:
lib3sat 1.2.6
Abhängigkeit zu mediathekxmlservice entfernt

68000a committed:
Phoenix Mediathek - 2.0.2
Abhängigkeit zu mediathekxmlservice entfernt

68000a committed:
libzdf 2.1.15
Sendungen nach A-Z: wenn möglich wird bei Serien der Serienname an den Titelanfang geschrieben

68000a committed:
Phoenix Mediathek - 2.0.3 / libkika 2.0.4 / libsr 2.0.4 / libmediathek3 1.5.14
- Beim Hinzufügen zu Favoriten gingen die Metadaten verloren
- Funktionen list() und getMetadata() zentralisiert

68000a committed:
libard 5.1.9
Sendersortierung korrigiert

68000a committed:
libard 5.1.10
Sendersortierung korrigiert

68000a committed:
libard 5.1.11
- Sendersortierung korrigiert
- Funktionen list() und getMetadata() zentralisiert

68000a committed:
addon.xml
Abhängigkeiten von libard korrigiert

68000a committed:
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.15
Vorbereitung für neue ARD-Mediathek

68000a committed:
libard 5.1.12
- Umlautproblem behoben: htmlParser.unescape() akzeptiert nur ASCII-Strings
- Vorbereitung für neue ARD-Mediathek

68000a committed:
addons.xml korrigiert

68000a committed:
libard 1.5.12
- Umlautproblem behoben: htmlParser.unescape() akzeptiert nur ASCII-Strings
- Vorbereitung für neue ARD-Mediathek

68000a committed:
libard 1.5.12
- Umlautproblem behoben: htmlParser.unescape() akzeptiert nur ASCII-Strings
- Vorbereitung für neue ARD-Mediathek

68000a committed:
libard 5.1.13
Heute war nicht mein Tag: Versionsnummer nochmals korrigiert

68000a committed:
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.16
Vorbereitung für neue ARD-Mediathek

68000a committed:
3sat Mediathek - 1.3.5 / lib3sat 1.2.7
Code vereinfacht

68000a committed:
BR Mediathek - 1.1.5 / libbr 2.0.9
Code vereinfacht

68000a committed:
Arte Mediathek - 1.0.8 / libarte 1.0.15
Code vereinfacht

68000a committed:
Das Erste Mediathek - 1.1.4 / libdaserste 1.1.5
Code vereinfacht

68000a committed:
Schönheitskorrektur

68000a committed:
WDR Mediathek - 1.2.4 / libwdr 1.2.6
Code vereinfacht

68000a committed:
Code vereinfacht (ohne Release)

68000a committed:
libmediathek3 1.5.17
"Sendungen nach Datum" funktionierte seit 1.März wg. Umlaut im Monatsnamen nicht mehr

purewasa committed:
libmediathek3 : Ersetze HTML-Entitäten in Untertiteln in lesbare Zeichen

68000a committed:
ARD Mediathek - 5.1.3 / libard 5.1.14
Implementierung "Neue ARD-Mediathek"

68000a committed:
Merge pull request #4 from purewasa/topic/translate-subtitles

libmediathek3 : Ersetze HTML-Entitäten in Untertiteln in lesbare Zeichen

68000a committed:
README-Datei aktualisiert

68000a committed:
README-Datei aktualisiert

68000a committed:
libard 5.1.15
- Sendungs-Untertitel funktionieren jetzt auch in der neuen ARD-Mediathek
- Sendungen nach A-Z glattgezogen
- Livestreams glattgezogen

68000a committed:
Mediatheken der Regionalsender MDR, NDR und SWR entfernt weil deren Inhalte nur noch über die ARD-Mediathek abrufbar sind

68000a committed:
libmediathek3 1.5.18
Vereinheitlichung der Listeneinträge bei "Sendungen nach Datum"

68000a committed:
3sat Mediathek - 1.3.6 / lib3sat 1.2.8
- Vereinheitlichung der Listeneinträge bei "Sendungen nach Datum"
- Code entflochten

68000a committed:
libard 5.1.16
Vereinheitlichung der Listeneinträge bei "Sendungen nach Datum"

68000a committed:
libarte 1.0.16
Vereinheitlichung der Listeneinträge bei "Sendungen nach Datum"

68000a committed:
libdaserste 1.1.6
Vereinheitlichung der Listeneinträge bei "Sendungen nach Datum"

68000a committed:
libwdr 1.2.7
Suche funktioniert wieder

68000a committed:
Schönheitskorrektur

68000a committed:
libard 5.1.17
Schönheitskorrektur bei "Sendungen nach Datum" / "Alle Sender"

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
Vorbereitungen für Matrix-Branch

68000a committed:
addons.xml.md5 angepasst

68000a committed:
Schönheitskorrekturen

68000a committed:
Quellcode vom AddOn-Verzeichnis verschoben

68000a committed:
Ursprüngliche repo_prep.py

68000a committed:
Standard-Repo in Gigathek-Krypton-Leia-1.0.0.zip umbenannt

68000a committed:
Änderungen aus Rev.453 wieder rückgängig gemacht

68000a committed:
Repository für Kodi-Matrix

68000a committed:
README-Datei aktualisiert

68000a committed:
README-Datei aktualisiert

68000a committed:
README-Datei aktualisiert

68000a committed:
Addon-Verzeichnis wieder entfernt

68000a committed:
repo_prep.py für die Generierung der GigathekMatrix-AddOns angepasst

68000a committed:
Gigathek Repository für Kodi 19 "Matrix"

68000a committed:
Gigathek Repository für Kodi 17 "Krypton" und Kodi 18 "Leia"

68000a committed:
libhr: Abhängigkeit zu BeautifulSoup4 vermerkt

68000a committed:
Anpassungen wg. Multiversions-Repo

68000a committed:
Anpassungen wg. Multiversions-Repo

68000a committed:
Multiversions-Repo 1.0.18

68000a committed:
Multiversions-Repo für Kodi 17 "Krypton", Kodi 18 "Leia" und Kodi 19 "Matrix"

68000a committed:
Weitere Anpassungen wegen Kodi Matrix

68000a committed:
Repo_prep.py: Spaces durch Tabs ersetzt

68000a committed:
KiKa Mediathek - 2.0.3 / libkika 2.0.5
Neue Einstellung "Adaptive Streams wenn möglich ignorieren"

68000a committed:
ARD Mediathek - 5.1.4 / libard 5.1.18 / libmediathek3 1.5.19
Download-Funktionalität hinzugefügt

68000a committed:
libard 5.1.19 / libmediathek3 1.5.20
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.21
Unicode-Problem in getSetting() unter Python 2.x behoben

68000a committed:
libard 5.1.20
Die Streaming-URLs wurden fälschlicherweise in Kleinbuchstaben umgewandelt

68000a committed:
libmediathek3 1.5.22
Neue libmediathek3 wg. Versions-Problemen im Multiversions-Repo

68000a committed:
libmediathek 1.5.23
Kleinere Verbesserungen

68000a committed:
libard 5.1.21
Code vereinfacht

68000a committed:
3sat Mediathek - 1.3.7 / lib3sat 1.2.9
Download-Funktionalität hinzugefügt

68000a committed:
ZDF Mediathek - 4.1.4 / ZDFtivi Mediathek - 1.0.2 / libzdf 2.1.16
Download-Funktionalität hinzugefügt

68000a committed:
libard 5.1.22
Kleinere Schönheitskorrekturen

68000a committed:
KiKa Mediathek - 2.0.4 / libkika 2.0.6
Download-Funktionalität hinzugefügt

68000a committed:
Schönheitskorrekturen

68000a committed:
ZDF Mediathek - 4.1.5 / libzdf 2.1.17
Implementierung "Mobile Version der ZDF Mediathek"

68000a committed:
libard 5.1.23
Exception "inconsistent use of tabs and spaces in indentation" behoben

68000a committed:
libard 5.1.24
Suche funktioniert wieder

68000a committed:
libmediathek3 1.5.24
Seltsames Verhalten der Suchfunktion unter Kodi 18 etwas abgemildert

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.25
Verhalten der Suchfunktion verbessert

68000a committed:
libmediathek3 1.5.26
Verhalten der Suchfunktion nochmals verbessert

68000a committed:
libard 5.1.25
Suche funktioniert jetzt auch mit Umlauten

68000a committed:
Repo-Update wg. Kodi 18.9

Commit:
repository link fix

68000a committed:
Merge pull request #5 from EdwarDDay/repository_link

repository link fix

juergenMSchmitz commented:
Hi jodo234 I had the same issue (gleiche problem).
For me the solution was to comment the following lines in libmediathek3listing.py out:

121     #if len(l) > 0:
122     #   type = l[0]['_type']
123         #        xbmc.log("type: " + type,level=xbmc.LOGNOTICE)
124     #   if type == 'video' or type == 'live' or type == 'date' or type == 'clip' or type == 'episode':
125     #       xbmcplugin.setContent( handle=int( sys.argv[ 1 ] ), content="episodes" )
126     #   elif type == 'shows' or type == 'season':
127     #       xbmcplugin.setContent( handle=int( sys.argv[ 1 ] ), content="tvshows" )
128     #   else:
129     #       xbmcplugin.setContent( handle=int( sys.argv[ 1 ] ), content="files" )

68000a committed:
libwdr 1.2.8
Menüpunkte "Neu" und "Sendungen nach Datum" entfernt: nicht mehr funktionsfähig

68000a committed:
README-Datei aktualisiert

68000a committed:
Schönheitskorrektur

68000a committed:
libwdr 1.2.9
Bugfix "Aktuelle Stunde"

68000a committed:
libmediathek3 1.5.27
Kleine Verbesserung der Download-Funktionalität

68000a committed:
libmediathek3 1.5.28
Notification anzeigen und Log-Eintrag erzeugen falls ein Download fehlschlug

68000a committed:
libard 5.1.26
Base-URL für Videos aus der Suchfunktion hat sich geändert

68000a committed:
Schönheitskorrekturen

68000a committed:
libard 5.1.27
Bugfix: nicht alle Livestreams der neuen ARD-Mediathek funktionierten

68000a committed:
libzdf 2.1.18
"Sendungen nach Datum" funktioniert wieder

68000a committed:
libarte 1.0.17
Sendungen nach A-Z funktioniert wieder

68000a committed:
libzdf 2.1.19
Classic-Version: "Meistgesehen" funktioniert wieder

68000a committed:
ARD Mediathek - 5.1.5 / libard 5.1.28
Zusammenführung der Classic- und der Mobile-Version der ARD Mediathek

68000a committed:
libard 5.1.29
Suche in der ARD Mediathek funktioniert wieder

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrektur

68000a committed:
ZDF Mediathek - 4.1.6 / ZDFtivi Mediathek - 1.0.3 / libzdf 2.1.20
Zusammenführung der Classic- und der Mobile-Version der ZDF Mediathek

68000a committed:
libard 5.1.30 / libzdf 2.1.21
Kompatibilität mit Python3 wiederhergestellt

68000a committed:
Arte Mediathek - 1.0.9 / libarte 1.0.18
Sendedatum bzw. Verfügbarkeit zum Plot hinzugefügt

68000a committed:
libarte 1.0.19
Hotfix "Suche"

68000a committed:
Schönheitskorrekturen

68000a committed:
Schönheitskorrektur

68000a committed:
Änderung aus Rev.526 wieder rückgängig gemacht

68000a committed:
Assets zur addon.xml hinzugefügt

68000a committed:
Assets zur addon.xml hinzugefügt

68000a committed:
Anpassungen für die neuen Matrix-Addons

68000a committed:
Phoenix Mediathek - 2.0.4
Phoenix Mediathek wieder funktionsfähig

68000a committed:
libmediathek3 1.5.29
Anpassungen für Kodi Matrix

68000a committed:
Versionsnummer der libmediathek3 angepasst

68000a committed:
libzdf 2.1.22
"Sendungen nach Datum / Mobile" funktioniert wieder

68000a committed:
KiKa Mediathek 2.0.5
KiKa Mediathek b.a.w. nicht verfügbar

68000a committed:
KiKa Mediathek - 2.0.6 / libkika 2.0.7
KiKa Mediathek wieder funktionsfähig

68000a committed:
ZDF Tivi: Versionsnummer korrigiert

68000a committed:
libarte 1.0.20
Suche: einige Suchbegriffe verursachten einen Absturz

68000a committed:
libarte 1.0.21
Schönheitskorrektur Info-Text

68000a committed:
Das Erste Mediathek seit 28.01.2021 nicht mehr verfügbar

68000a committed:
Das Erste Mediathek - 1.1.5
Das Erste Mediathek komplett in die ARD-Mediathek umgezogen
siehe: https://www.daserste.de/specials/service/app-102.html

68000a committed:
libwdr 1.2.10
"Sendungen A-Z": Exception bei Buchstaben A und D behoben

68000a committed:
Mediathek "Das Erste" entfernt weil die Inhalte nur noch über die ARD-Mediathek abrufbar sind

68000a committed:
Mediathek "Das Erste" entfernt weil die Inhalte nur noch über die ARD-Mediathek abrufbar sind

68000a committed:
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.30
Anpassungen für Kodi Nexus

68000a committed:
libzdf 2.1.23
Rubriken/Classic funktioniert wieder

68000a committed:
libbr 2.0.10
Anzeige der Titel verbessert

68000a committed:
libard 5.1.31
Mobile-Version der ARD Mediathek wieder funktionsfähig

68000a committed:
libard 5.1.32
Schönheitskorrektur

68000a committed:
libbr 2.0.11
Korrektur: Meistgesehen -> Neu

68000a committed:
Phoenix Mediathek 2.0.5
Phoenix Mediathek ist in der ARD Mediathek aufgegangen

68000a committed:
Phoenix Mediathek ist in der ARD Mediathek aufgegangen

68000a committed:
Phoenix Mediathek 2.0.6
Phoenix Mediathek wieder da :-)

68000a committed:
libard 5.1.33
Suche in der ARD Mediathek funktioniert wieder

68000a committed:
libzdf 2.1.24
Meistgesehen funktioniert nicht mehr (HTTP-Error 410 - Gone)

68000a committed:
libmediathek3 1.5.31
Livestreams sind jetzt auch unter Kodi Matrix zurückspulbar

68000a committed:
libarte 1.0.22
Sendungen nach Datum: Uhrzeit wird wieder angezeigt

68000a committed:
libard 5.1.34
Livestream "Das Erste" funktioniert wieder

68000a committed:
libarte 1.0.23
Meistgesehen: TV-Serien wurden nicht berücksichtigt

68000a committed:
libard 5.1.35
Classic-Mediathek z.Zt. nicht funktionsfähig

68000a committed:
ARTE-Mediathek entfernt weil die Umstellung auf die neuen APIs zu aufwendig wäre.
Es wird empfohlen, die ARTE-Mediathek aus dem Nerdsrepo zu verwenden: https://repo.kodinerds.net

68000a committed:
Phoenix-Mediathek entfernt weil das verwendete API nicht mehr verfügbar ist.
Es wird empfohlen, als Ersatz die ARDundZDF-Mediathek aus dem Nerdsrepo zu verwenden: https://repo.kodinerds.net

68000a committed:
Mediathek des Saarländischen Rundfunks entfernt weil die Inhalte nur noch über die ARD-Mediathek abrufbar sind

68000a committed:
Mediathek des Hessischen Rundfunks entfernt weil die Inhalte nur noch über die ARD-Mediathek abrufbar sind

68000a committed:
ARTE Mediathek - 1.0.10 / libarte 1.0.25
ARTE-Mediathek reaktiviert, alle Streams können wieder abgespielt werden

68000a committed:
Schönheitskorrektur

68000a committed:
libmediathek3 1.5.32
Kleiner Bugfix bei Livestreams

68000a committed:
libmediathek3 1.5.33
Änderungen aus 1.5.32 wieder rückgängig gemacht

68000a committed:
libmediathek3 1.5.34 / lib3sat 1.2.10 / libard 5.1.36 / libarte 1.0.26 / libbr 2.0.12 / libkika 2.0.8 / libwdr 1.2.11 / libzdf 2.1.25
Umstellung auf neues Layout zur Anzeige der Videos

68000a committed:
libard 5.1.37
Livestream "Das Erste" und andere funktionieren wieder

68000a committed:
libzdf 2.1.26
Livestream-URLs werden gecached (analog ARD-Addon)

68000a committed:
libard 5.1.38
Wiedergabe und Download über die Suche funktioniert wieder

68000a committed:
libard 5.1.39
Wiedergabe und Download über die Suche funktioniert wieder

68000a committed:
libard 5.1.40
Sendungen nach A-Z funktioniert wieder

68000a committed:
libbr 2.0.13
Sporadische Exception bei "Sendungen nach Datum" behoben

68000a committed:
libbr 2.0.14
Weitere sporadische Exception bei "Sendungen nach Datum" behoben

68000a committed:
libard 5.1.41
Livestream "Das Erste" (und andere) funktioniert wieder

68000a committed:
libard: Sendungen nach A-Z funktioniert wieder

68000a committed:
ARD Mediathek - 5.1.6 / libard 5.1.42
Sendungen nach A-Z funktioniert wieder

68000a committed:
ZDF Mediathek - 4.1.7 / libzdf 2.1.27
Mobile-Version: Sendedatum und/oder Abrufbar-bis-Datum hinzugefügt

68000a committed:
Fanart in allen Mediatheken aktiviert

68000a committed:
libbr 2.0.15
Sortierung geändert: "BR" jetzt vor "ARD Alpha"

68000a committed:
libard: Problem mit NDR-Livestream behoben

68000a committed:
libmediathek3 1.5.35 / libard 5.1.43 / libzdf 2.1.28
Stream-Position (Uhrzeit) einblenden bei vor-/zurückspulen im Livestream

68000a committed:
libmediathek3 1.5.36
Keine Stream-Position einblenden bei weniger als 20 Sekunden vor-/zurückspulen

68000a committed:
Schönheitskorrektur

68000a committed:
Schönheitskorrekturen

68000a committed:
libbr 2.0.16
"Sendungen nach Datum" etwas robuster gemacht

68000a committed:
ARTE Mediathek - 1.0.12 / libarte 1.0.27
In den Einstellungen der ARTE Mediathek kann das bevorzugte Stream-API ausgewählt werden (Default / OPAv3)

68000a committed:
libard 5.1.44
Sendungen nach Datum: Umstellung auf anderes API

68000a committed:
Schönheitskorrekturen

68000a committed:
libwdr 1.2.12
Schönheitskorrektur

68000a committed:
libbr 2.0.17
Bugfix: "Sendungen nach Datum" listete auch Sendungen der Vortage auf

68000a committed:
libbr 2.0.17
Bugfix: "Sendungen nach Datum" listete auch Sendungen der Vortage auf

68000a committed:
libarte 1.0.28
Umstellung von OPA-API auf EMAC-API => "Meistgesehen", "Suche" und Unterordner funktionieren wieder

68000a committed:
libzdf 2.1.29
Das ZDF-API hat eine neue URL

68000a committed:
libarte 1.0.29
Stream-API "OPAv3" funktioniert wieder

68000a committed:
ARTE Mediathek - 1.0.13
Download möglich falls Stream-API auf OPAv3 eingestellt ist

68000a committed:
libard 5.1.45
Abspielen von Videos aus der Suche heraus funktioniert wieder

68000a committed:
ZDF Mediathek - 4.1.9 / libzdf 2.1.30
Livestreams wahlweise aus Datei 'livestream.json' lesen oder via API beziehen
Datei 'livestream.json' wahlweise in special://home/addons oder in special://profile/addon_data suchen

68000a committed:
libzdf 2.1.30
Livestreams wahlweise aus Datei 'livestream.json' lesen oder via API beziehen
Datei 'livestream.json' wahlweise in special://home/addons oder in special://profile/addon_data suchen

68000a committed:
libmediathek3 1.5.37
In Kodi 19, the validatePath function has been moved from xbmc to xbmcvfs

68000a committed:
ZDF-Tokengrabber angepasst

68000a committed:
Future

68000a committed:
lib3sat 1.2.11
Subdirectories werden jetzt (meistens) korrekt erkannt

68000a committed:
Anpassungen wg. script.module.futures (Builtin ab Python 3)

68000a committed:
lib3sat 1.2.12
Subdirectories werden jetzt (meistens) korrekt erkannt

68000a committed:
Logos der Mediatheken hinzugefügt

68000a committed:
Obskure Fehlermeldungen im Logfile von Kodi Omega vermeiden

68000a committed:
Schönheitskorrektur

68000a committed:
libarte 1.0.30
Sendungen nach Datum funktioniert wieder

68000a committed:
libmediathek3 1.5.38
Bugfix Livestream-Position für Livestreams bei denen sich die Gesamtzeit während des Anschauens verlängert

68000a committed:
libmediathek3 1.5.39
Der Bugfix aus Version 1.5.38 wurde nochmals verbessert

68000a committed:
repository.gigathek 1.0.21
Addon-ID changed from "Gigathek" to "repository.gigathek"

68000a committed:
Revert "repository.gigathek 1.0.21"

This reverts commit 2611117be60c71611672f9dca52bb94454986b93.

68000a committed:
libard 5.1.46
Das API der ARD hat sich geändert. Abspielen von Videos funktioniert jetzt wieder

68000a committed:
libard 5.1.47
Streams mit Audio-Deskription bei der automatischen Stream-Auswahl vermeiden

68000a committed:
Anpassung README.md

68000a committed:
Schönheitskorrektur

kujub committed:
code: Use https instead of http

68000a committed:
libmediathek3 1.5.40
Schönheitskorrektur bei Anzeige der Livestream-Position

68000a committed:
Changelog-Texte von ANSI nach UTF-8 konvertiert

68000a committed:
Merge pull request #6 from kujub/https

code: Use https instead of http

68000a committed:
libard 5.1.48 / libarte 1.0.31
Use https instead of http

68000a committed:
Merge branch 'master' of github.com:68000a/Gigathek

68000a committed:
libkika 2.0.10
Behoben: "Sendungen Neu" machte ab und an Probleme

68000a committed:
libzdf 2.1.31
Behoben: sporadische Exception "list index out of range" in libzdftokengrabber.py

68000a committed:
libwdr 1.2.14
Menüpunkt "Suche" wurde entfernt weil nicht mehr funktionsfähig

68000a committed:
libard 5.1.49
Die automatische Stream-Auswahl verwendet bei Medien mit explizit ausgewiesener Audio-Despription ab jetzt den korrekten Stream mit Audio-Despription. Dies betrifft sowohl Abspielen als auch Download.
